service/docs/source/geopmagent_policy.schema.json

```diff
@@ -0,0 +1,9 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "https://geopm.github.io/geopmagent_policy.schema.json",
+    "title": "GEOPM agent policy",
+    "type": "object",

```

Comment:
How does this handle differences in the way the policy is specified based on the selected Agent?

Reply:
> How does this handle differences in the way the policy is specified based on the selected Agent?

This won't offer deeper name checking (i.e. it won't allow validating that the policy parameter names are correct for a particular agent). This simply specifies that the JSON data must be a set of key-value pairs, where the values can be strings or integers. This is consistent with what the code validates (i.e. FilePolicy is not checking the names).

This is reasonable for a general policy schema. Otherwise, we would have to have a policy schema for each agent (which we could do, I guess).